git_theta/updates/dense.py

```diff
@@ -18,11 +18,19 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     async def apply(self, param_metadata, param_keys, *args, **kwargs) -> Parameter:
-        logging.debug(f"Reading Dense update for {'/'.join(param_keys)}")
-        return await self.read(param_metadata)
+        param_name = "/".join(param_keys)
+        logging.debug(f"Reading Dense update for {param_name}")
+        tensor = await self.read(param_metadata)
+        logging.debug(f"Read Dense update for {param_name}")
```

Comment:
Nit: "Finished reading" since "Read" can be read in the present or past tense?

Reply:
Fixed